apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPoolImpl.cs

```diff
@@ -38,10 +41,13 @@ internal class SessionPoolImpl : IPriorityListItem<SessionPoolImpl>
 
         public SessionPoolKey Key { get; }
 
-        internal SessionPoolImpl(SessionPoolKey key)
+        internal SessionPoolImpl(SessionPoolKey key, SessionPoolOptions options)
         {
+            GaxPreconditions.CheckNotNull(options, nameof(options));
+            _options = options;
```

Comment:
I believe this can just be `_options = GaxPreconditions.CheckNotNull(options, nameof(options));`